tests/min-reporter.cjs

```diff
@@ -0,0 +1,10 @@
+const { reporters } = require('mocha');
+
+class EvenMoreMin extends reporters.Base {
+  constructor(runner) {
+    super(runner);
+    runner.once('end', () => this.epilogue());
```

Comment:
I think that it would be more secure to use the constants directly:

```suggestion
const { constants } = require("mocha/lib/runner")

class EvenMoreMin extends reporters.Base {
  constructor(runner) {
    super(runner);
    runner.once(constants.EVENT_RUN_END, () => this.epilogue());
```

Reply:
We are thinking the same 😅